x-pack/metricbeat/module/azure/compute_vm/_meta/data.json

```diff
@@ -125,28 +97,10 @@
     },
     "host": {
         "cpu": {
-            "pct": 0.01585
+            "pct": 0.09747
         },
-        "disk": {
-            "read": {
-                "bytes": 4019543.24
-            },
-            "write": {
-                "bytes": 16739967.27
-            }
-        },
-        "id": "/subscriptions/7657426d-c4c3-44ac-88a2-3b2cd59e6dba/resourceGroups/obs-test/providers/Microsoft.Compute/virtualMachines/perfmon-test",
-        "name": "perfmon-test",
-        "network": {
-            "in": {
-                "bytes": 1806936,
-                "packets": 1501534
-            },
-            "out": {
-                "bytes": 3633130,
-                "packets": 1647640
-            }
-        }
+        "id": "d5d9444a-1964-4d23-9c62-5463ecb16fe0",
+        "name": "obslinux"
```

Comment:
Uh oh, we are missing the rest of the host fields.

Reply:
@kaiyan-sheng , the values used to map those host fields are not retrieved in this event, most likely the next one containing those values will. as you can see only `percentage_cpu` is contained in the metrics.